mongodbatlas/data_source_mongodbatlas_search_indexes.go

```diff
@@ -65,17 +62,14 @@ func dataSourceMongoDBAtlasSearchIndexesRead(ctx context.Context, d *schema.Reso
 		return diag.Errorf("project_id, cluster_name, database and collection_name must be configured")
 	}
 
-	options := &matlas.ListOptions{
-		PageNum:      d.Get("page_num").(int),
-		ItemsPerPage: d.Get("items_per_page").(int),
-	}
```

Comment:
still pending what to do with page_num and items_per_page, but please review the rest of the PR

Reply:
will be done in another PR to better track changlog